pages/emergency-info.tsx

```diff
@@ -0,0 +1,307 @@
+import { PaperClipIcon } from "@heroicons/react/solid";
+import Link from "next/link";
+import { useRouter } from "next/router";
+import React from "react";
+import { FormProvider, SubmitHandler, useForm } from "react-hook-form";
+import { useIntl } from "react-intl";
+import { colorMap } from "../components/Button";
+import { Input, Submit, TextArea } from "../components/formComponents";
+import FormFooter from "../components/FormFooter";
+import Layout from "../components/Layout";
+import Page, { LeftSection, RightSection } from "../components/Page";
+import Button from "../components/Button";
+
+type FormValues = {
+  currentEmail: string;
+  newEmail: string;
+  password: string;
+};
+const bold = (msg: string): React.ReactNode => (
```

Comment:
Consider changing the name of this to `strong` and using <strong> tags instead of css within the function itself.